playgrounds/vue-3/index.html

```diff
@@ -6,17 +6,25 @@
 
     <title>Vue Tester App</title>
 
-    <link
+    <!-- <link
       href="https://dhqbrvplips7x.cloudfront.net/common-ui-docs/genesys-webcomponents/3.59.0-715/genesys-webcomponents/genesys-webcomponents.css"
       rel="stylesheet"
     />
 
     <script
       type="module"
       src="https://dhqbrvplips7x.cloudfront.net/common-ui-docs/genesys-webcomponents/3.59.0-715/genesys-webcomponents/genesys-webcomponents.esm.js"
```

Comment:
Just checking this comment is supposed to be left in?

Reply:
Yeah I left it in just so that it's easier to comment back in if we want to test a certain version in the playground